charts/gateway-helm/values.tmpl.yaml

```diff
@@ -66,6 +66,10 @@ createNamespace: false
 
 kubernetesClusterDomain: cluster.local
 
+# Certgen is used to generate the certificates required by EnvoyGateway.
+# If you want to construct a custom certificate, you can generate a custom certificate through Cert-Manager before installing EnvoyGateway.
+# Certgen will not overwrite the custom certificate.
+# WARNING: Please do not manually modify `values.yaml` to disable certgen, which may cause EnvoyGateway OIDC,OAuth2,etc. to not work as expected.
```

Comment:
```suggestion
# WARNING: Please do not manually modify `values.yaml` to disable certgen, it may cause EnvoyGateway OIDC,OAuth2,etc. to not work as expected.
```